packages/webhooks/tasks/sendWebhook.spec.js

```diff
@@ -48,37 +85,150 @@ const SendWebhookTests = (appName, actorsInitializer, userCreator, userDestroyer
         })
-        it('Must correctly send requests and update subscription state', async () => {
-            const [hook] = await createTestWebhook(actors.admin, actors.admin.user)
-            const [subscription] = await createTestWebhookSubscription(actors.admin, hook, {
-                syncedAt: dayjs(firstUser.createdAt).subtract(10, 'millisecond'),
-                url: STABLE_URL,
-            })
-            expect(subscription).toHaveProperty('syncedAt')
-            const initialSyncTime = dayjs(subscription.syncedAt)
-            await sendWebhook.delay.fn(subscription.id)
+        afterEach(() => {
+            STABLE_CALLS.splice(0)
+        })
+        describe('Must correctly send requests and update subscription state', () => {
+            describe('with an "operations" equal to', async () => {
+                const cases = [
+                    [
+                        'null (default)',
+                        null,
+                        () => {
+                            // NOTE: DELETED USER MUST BE IN SENT DATA
+                            expect(STABLE_CALLS[0]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: firstUser.id }),
+                                expect.objectContaining({ id: deletedUser.id }),
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS[1]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: firstUser.id }),
+                            ]))
+                            expect(STABLE_CALLS[2]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS.flat()).toHaveLength(5)
+                        },
+                    ],
+                    [
+                        'create',
+                        { create: true, update: false, delete: false },
+                        () => {
+                            // NOTE: DELETED USER MUST BE IN SENT DATA
+                            expect(STABLE_CALLS[0]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: firstUser.id }),
+                                expect.objectContaining({ id: deletedUser.id }),
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS.flat()).toHaveLength(3)
+                        },
+                    ],
+                    [
+                        'update',
+                        { create: false, update: true, delete: false },
+                        () => {
+                            // NOTE: DELETED USER MUST BE IN SENT DATA
+                            expect(STABLE_CALLS[0]).toEqual(expect.arrayContaining([
```

Comment:
This call should not exist for update-only.
 
in beforeAll we create 3 users and delete 1 of them....So it `3 x create` + `1 x delete`. Why they're in update hook?